packages/shared/edge-utils/src/app.ts

```diff
@@ -0,0 +1,171 @@
+import { AutoExperimentVariation, GrowthBook, setPolyfills } from "@growthbook/growthbook";
+import { Context } from "./types";
+import { getUserAttributes } from "./attributes";
+import { getCspInfo, injectScript } from "./inject";
+import { applyDomMutations } from "./domMutations";
+import redirect from "./redirect";
+import { getRoute } from "./routing";
+import { EdgeStickyBucketService } from "./stickyBucketService";
+
+export async function edgeApp(
+  context: Context,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  req: any,
```

Comment:
we might want to set these to unknown so then we can having better control over the types after the fact